protocol/contracts/zaps/GenericZap.sol

```diff
@@ -37,7 +37,7 @@ contract GenericZap is ZapBase {
 
   constructor (
```

Comment:
I thought we removed these from the repo :-)

Reply:
:-)